src/taipy/core/config/checkers/_scenario_config_checker.py

```diff
@@ -115,10 +115,23 @@ def _check_additional_dns_not_overlapping_tasks_dns(self, scenario_config_id: st
     def _check_tasks_in_sequences_exist_in_scenario_tasks(
         self, scenario_config_id: str, scenario_config: ScenarioConfig
     ):
-        scenario_tasks = scenario_config.tasks
-        for sequence in scenario_config.sequences.values():
-            for task in sequence:
-                if task not in scenario_tasks:
+        scenario_task_ids = set()
+        for task_config in scenario_config.tasks:
+            if isinstance(task_config, TaskConfig):
+                scenario_task_ids.add(task_config.id)
+            else:
+                return
```

Comment:
Why do we return here?
- If it is not a TaskConfig, that's an error to collect, no?
- If the error is checked somewhere else, we should continue and not return. No?

Reply:
That's actually for the unit tests only. Some of the tests we are catching the SystemExit error and keep on running the code. As a result, the rest of the code will be run and result in failure. In reality, the condition will not fall into the False case as if it does, it'd have been caught by `_check_task_configs` in ScenarioConfig checkers